### DISCUSSION TITLE
Table vwmonthdata interpretation

### DISCUSSION THREAD

**Discussion by PRoomberg** [SBFspot/SBFspot#704] [Q&A] [answered]
Installed SBFspot today and started getting data from my 2 SMA inverters.
Everything looks ok, except for the table vwmonthdata, which confuses me.

It shows the TotalYield value for the inverter as it as at the end of a day.
But the TimeStamp field includes the time 00:00:00, which is the beginning of the day. 

My goal is to get the TotalYield for the inverter as it is at the beginning of the day.

**Comment by sbf-** [comment, marked as answer]
I don't see any time 00:00:00
It is indeed EOD yield (inverter resets counter at midnight - at least, mine does)

```
pi@sbfspot:~ $ cd smadata/
pi@sbfspot:~/smadata $ sqlite3 SBFspot.db 
SQLite version 3.27.2 2019-02-25 16:06:06
Enter ".help" for usage hints.
sqlite> select * from vwMonthData limit 5;
2024-07-11|SB4000TL - 2100276197|SB 4000TL-20|2100276197|55897896|17429
2024-07-10|SB4000TL - 2100276197|SB 4000TL-20|2100276197|55880467|17433
2024-07-09|SB4000TL - 2100276197|SB 4000TL-20|2100276197|55863034|12649
2024-07-08|SB4000TL - 2100276197|SB 4000TL-20|2100276197|55850385|10932
2024-07-07|SB4000TL - 2100276197|SB 4000TL-20|2100276197|55839453|11658
sqlite> 
```

**Reply by PRoomberg** [reply]
Thanks for confirming it is EOD yield.

I'm using MySQL as a database and my queries return time values as well:

```
MySQL [SBFspot]> select Timestamp, TotalYield, DayYield from vwmonthdata order by timestamp desc;
+---------------------+------------+----------+
| TimeStamp           | TotalYield | DayYield |
+---------------------+------------+----------+
| 2024-07-11 00:00:00 |   19163745 |    11688 |
| 2024-07-11 00:00:00 |   17565802 |     8923 |
| 2024-07-10 00:00:00 |   19152057 |     8631 |
| 2024-07-10 00:00:00 |   17556879 |     7808 |
| 2024-07-09 00:00:00 |   19143426 |    10519 |
| 2024-07-09 00:00:00 |   17549071 |     8449 |
| 2024-07-08 00:00:00 |   19132907 |     7347 |
| 2024-07-08 00:00:00 |   17540622 |     5809 |
| 2024-07-07 00:00:00 |   19125560 |     8527 |
```

I've also fetched the monthdata with parameter -am144 to see if I could go back to day 1. 
That results in time values of 23:00:00 or 22:00:00

```
| 2023-01-19 00:00:00 |   16456252 |     1061 |
| 2023-01-17 23:00:00 |   16455191 |     1516 |
| 2023-01-16 23:00:00 |   16453675 |     1294 |
| 2023-01-15 23:00:00 |   16452381 |     1046 |
```

and

```
| 2021-12-02 00:00:00 |   14419350 |      592 |
| 2021-12-01 00:00:00 |   14418758 |      480 |
| 2021-11-29 22:00:00 |   14418278 |       48 |
| 2021-11-28 22:00:00 |   14418230 |      592 |
| 2021-11-27 22:00:00 |   14417638 |      722 |
```

and 

```
| 2021-06-21 23:00:00 |   13616862 |     3212 |
| 2021-06-21 00:00:00 |   13613650 |     4558 |
| 2021-06-19 23:00:00 |   13609092 |    10752 |
```

I know I've had issues with the inverter loosing the correct time, so that might be why I'm getting those values.

**Comment by sbf-** [comment]
You can always double check with [Sunnyexplorer](https://www.sma.de/en/products/monitoring-control/sunny-explorer)

Also missing serial number in your outputs since you have 2 inverters

**Comment by PRoomberg** [comment]
Thanks for the tip, but until yesterday I was fetching data using Sunny Explorer and uploading that to PVoutput.
The CSV files that Sunny Explorer exports show the value for TotalYield at the end of the day.
That's how I noticed this difference.

The output above is data from only 1 inverter, but I left the serial number out.
